### DISCUSSION TITLE
v0.12.9

### DISCUSSION THREAD

**Discussion by david-rfai** [RapidFireAI/rapidfireai#145] [Announcements]
## What's Changed
* unified automl files by @david-rfai in https://github.com/RapidFireAI/rapidfireai/pull/111
* Improve FiQA RAG Colab notebook explanations by @kamran-rapidfireAI in https://github.com/RapidFireAI/rapidfireai/pull/130
* feat: add Open in Colab buttons to tutorial notebooks by @kamran-rapidfireAI in https://github.com/RapidFireAI/rapidfireai/pull/131
* mlflow + trackio logging by @humaira-rf in https://github.com/RapidFireAI/rapidfireai/pull/129
* Fix metric logger options by @david-rfai in https://github.com/RapidFireAI/rapidfireai/pull/135
* fixed invalid chunk_id issue for last chunk by @pradyumna-rfai in https://github.com/RapidFireAI/rapidfireai/pull/136
* Fixed fair round robin scheduler to also consider num_epochs_completed by @pradyumna-rfai in https://github.com/RapidFireAI/rapidfireai/pull/137
* Add MLFLow for Evals with combined frontend by @david-rfai in https://github.com/RapidFireAI/rapidfireai/pull/138
* Bugfix/tensorboard experiments by @david-rfai in https://github.com/RapidFireAI/rapidfireai/pull/139
* Revert "Add MLFLow for Evals with combined frontend" by @david-rfai in https://github.com/RapidFireAI/rapidfireai/pull/142
* Release/v0.12.9 by @david-rfai in https://github.com/RapidFireAI/rapidfireai/pull/144

**Full Changelog**: https://github.com/RapidFireAI/rapidfireai/compare/v0.12.8...v0.12.9

<hr /><em>This discussion was created from the release <a href='https://github.com/RapidFireAI/rapidfireai/releases/tag/v0.12.9'>v0.12.9</a>.</em>
